src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java

```diff
@@ -179,6 +179,43 @@ public String getWikiGlobalURL() {
         .replaceFirst("/$", "");
   }
 
+  /**
+   * Gets env-specific url for a service
+   * @param serviceName normalized service name as used in its url, e.g. spam-wiki-review
+   */
+  public String getUrlForService(String serviceName) {
+    return getUrlForService(serviceName, false);
+  }
+
+  /**
+   * Gets env-specific url for a service
+   * @param serviceName normalized service name as used in its url, e.g. spam-wiki-review
+   */
+  public String getUrlForService(String serviceName, boolean addWWW) {
+    return getUrlForService(serviceName, addWWW, envType);
+  }
+
+  /**
+   * Gets env-specific url for a service
+   * @param serviceName normalized service name as used in its url, e.g. spam-wiki-review
+   */
+  public String getUrlForService(String serviceName,  boolean addWWW, EnvType envType){
+    HttpUrl.Builder urlBuilder = new HttpUrl.Builder();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "urlBuilder" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1481)